Tools-Override/tests.targets

```diff
@@ -262,4 +262,6 @@
   <Target Name="BuildAndTest" DependsOnTargets="Build;Test" />
   <Target Name="RebuildAndTest" DependsOnTargets="Rebuild;Test" />
 
+  <!-- This helps collect crash dumps and requires python installed -->
```

Comment:
I think this is the only part that needs to be added. The other dumpling files shouldn't need to be included in Tools-Override they should still come from the BuildTools package.